lib/plugins/aws/info/getStackInfo.js

```diff
@@ -57,6 +44,24 @@ module.exports = {
       }
 
       return BbPromise.resolve();
+    })
+    .then(() => this.provider.getAccountId())
+    .then((accountId) => {
+      const stage = this.provider.getStage();
+      const region = this.provider.getRegion();
+
+      this.gatheredData.info.functions = [];
+
+      this.serverless.service.getAllFunctions().forEach((func) => {
+        const functionInfo = {};
+        const name = `${this.serverless.service.service}-${stage}-${func}`;
+        const arn = `arn:aws:lambda:${region}:${accountId}:function:${name}`;
```

Comment:
Thanks for pointing into that direction @ryansb ! I'll look into it.
Will be more reliable when fetching through this API